### PR TITLE
Replace 'on offer' with 'to accept' in jobs requiring outfits, and update those jobs

### PR DESCRIPTION
**Pull request by bene-dictator**
## Summary
Changes all 'on offer' for 'to accept' in jobs that require Brigs or Luxury Accommodations. They will now show up in the jobs screen even if you don't have those outfits installed. The descriptions of the jobs now give you a clue to what outfit you might need to take the job. 

## Testing Done
I need to check if the descriptions of the missions don't go over the text box. I'll do this soon.

### CODE REVIEW

**Comment by warp-core** [comment]
None of these mention the need for these outfits in the description.
Appropriate adjustments should be made to the descriptions or `to offer` should be used instead of `to accept`.

**Comment by bene-dictator** [comment]
> None of these mention the need for these outfits in the description. Appropriate adjustments should be made to the descriptions or `to offer` should be used instead of `to accept`.
> (Al;so, the tests fail because you need to do `has <condition>`.)

Fair enough... I will do that. Before I had used "blocked" but I didn't think that would work on a job.

File: data/human/paradise world jobs.txt [hunk 1]
```diff
@@ -453,13 +453,13 @@ mission "Paradise Job: Debtors"
 	name "Debtors to <planet>"
 	job
 	repeat
-	description `These <bunks> poor souls have bankrupted themselves attempting to live beyond their means. Due to media scrutiny on the issue, the local government will pay you <payment> to transport them to <destination> where they can "make something useful of themselves."`
+	description `These <bunks> poor souls have bankrupted themselves attempting to live beyond their means. Due to media scrutiny on the issue, the local government will pay you <payment> to transport them in a cell to <destination> where they can "make something useful of themselves."`
```
Comment (Amazinite) [inline data/human/paradise world jobs.txt:456]: This is too long (and was probably already too long before).
image
image

File: data/human/paradise world jobs.txt [hunk 2]
```diff
@@ -429,13 +429,13 @@ mission "Paradise Job: Parolees"
 	name "Parolees to <planet>"
 	job
 	repeat
-	description `These <bunks> non-violent repeat offenders have accepted longer community service sentences that better use their skills in return for serving their time on the Paradise World of <destination>. Transport them to their new home to carry out their service. The payment is <payment>.`
+	description `These <bunks> non-violent repeat offenders have accepted longer community service sentences that better use their skills in return for serving their time on the Paradise World of <destination>. Transport them in a cell to their new home where they will carry out their service. The payment is <payment>.`
```
Comment (Amazinite) [inline data/human/paradise world jobs.txt:432]: Ditto.
image

File: data/human/pirate jobs.txt [hunk 3]
```diff
@@ -1665,15 +1665,15 @@ mission "Slave Transport [0]"
 	name "Transfer slaves to <planet>"
 	job
 	repeat
-	description "A pirate on <planet> has bought slaves from a local warlord. Transfer these <bunks> slaves to <destination> for <payment>. Slavery is punishable by death throughout the galaxy, so this may be a dangerous job to take."
+	description "A pirate on <planet> has bought slaves from a local warlord. Transfer these <bunks> slaves to <destination> for <payment>. To keep them from making any trouble, you'll need suitable facilities to transport them. Slavery is punishable by death throughout the galaxy, so this may be a dangerous job to take."
```
Comment (Amazinite) [inline data/human/pirate jobs.txt:1668]: image

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Comment by bene-dictator** [comment]
Thanks for testing those! Some of them might just need changing from `<destination>` to `<planet>`, but I'll try to change the text first because `<destination>` seems to be used more here than `<planet>` for mission summaries.

**Comment by bene-dictator** [comment]
Screenshot_20221228_164107
Just had this mission... Farpoint is a notoriously exotic planet.

**Review by Amazinite — APPROVED** [review]

**Pull request merged**